server/tests/hybrid/test_hybrid.py

```diff
@@ -720,6 +720,49 @@ def test_hybrid_manifest_validation_few_non_cvr_ballots(
     )
 
 
+def test_hybrid_manifest_validation_too_many_cvr_votes(
+    client: FlaskClient,
+    election_id: str,
+    jurisdiction_ids: List[str],
+    election_settings,  # pylint: disable=unused-argument
+    manifests,  # pylint: disable=unused-argument
+    cvrs,  # pylint: disable=unused-argument
+):
+    set_logged_in_user(client, UserType.AUDIT_ADMIN, DEFAULT_AA_EMAIL)
+    contests = [
+        {
+            "id": str(uuid.uuid4()),
+            "name": "Contest 1",
+            "isTargeted": True,
+            "choices": [
+                {"id": str(uuid.uuid4()), "name": "Choice 1-1", "numVotes": 13},
+                {"id": str(uuid.uuid4()), "name": "Choice 1-2", "numVotes": 10},
+            ],
+            "numWinners": 1,
+            "votesAllowed": 2,
+            "jurisdictionIds": jurisdiction_ids[:2],
+        },
```

Comment:
What about this setup triggers the error? For example, is it the fact that 13 is greater than 2 (votes allowed)?

Reply:
It's that 13 is less than the total votes for Choice 1-1 in the CVR. I can try to make that a bit more clear